_meetups/2024-02-20-meetup.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: meetup
+title: "Programmable OS Command Palettes"
+slides:
+video:
+speakers:
+  - name: Kevin Kipp
+    title:
+    avatar: https://avatars.githubusercontent.com/u/8732191?v=4
+    bio:
+    email:
+    homepage:
```

Comment:
Can we add https://kevinkipp.com here? 😃

Reply:
fixed!